apis/v1alpha1/record_set.go

```diff
@@ -0,0 +1,483 @@
+	// latency resource record set that has the lowest latency between the end user
+	// and the associated Amazon EC2 Region. Route 53 then returns the value that
+	// is associated with the selected resource record set.
+	//
+	// Note the following:
+	//
+	//   - You can only specify one ResourceRecord per latency resource record
+	//     set.
+	//
+	//   - You can only create one latency resource record set for each Amazon
+	//     EC2 Region.
+	//
+	//   - You aren't required to create latency resource record sets for all Amazon
+	//     EC2 Regions. Route 53 will choose the region with the best latency from
+	//     among the regions that you create latency resource record sets for.
+	//
+	//   - You can't create non-latency resource record sets that have the same
+	//     values for the Name and Type elements as latency resource record sets.
+	Region *string `json:"region,omitempty"`
+	// Information about the resource records to act upon.
+	//
+	// If you're creating an alias resource record set, omit ResourceRecords.
+	ResourceRecords []*ResourceRecord `json:"resourceRecords,omitempty"`
+	// Resource record sets that have a routing policy other than simple: An identifier
+	// that differentiates among multiple resource record sets that have the same
+	// combination of name and type, such as multiple weighted resource record sets
+	// named acme.example.com that have a type of A. In a group of resource record
+	// sets that have the same name and type, the value of SetIdentifier must be
+	// unique for each resource record set.
+	//
+	// For information about routing policies, see Choosing a Routing Policy (https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/routing-policy.html)
+	// in the Amazon Route 53 Developer Guide.
+	SetIdentifier *string `json:"setIdentifier,omitempty"`
+	// The resource record cache time to live (TTL), in seconds. Note the following:
+	//
+	//   - If you're creating or updating an alias resource record set, omit TTL.
+	//     Amazon Route 53 uses the value of TTL for the alias target.
+	//
+	//   - If you're associating this resource record set with a health check (if
+	//     you're adding a HealthCheckId element), we recommend that you specify
+	//     a TTL of 60 seconds or less so clients respond quickly to changes in health
+	//     status.
+	//
+	//   - All of the resource record sets in a group of weighted resource record
+	//     sets must have the same value for TTL.
+	//
+	//   - If a group of weighted resource record sets includes one or more weighted
+	//     alias resource record sets for which the alias target is an ELB load balancer,
+	//     we recommend that you specify a TTL of 60 seconds for all of the non-alias
+	//     weighted resource record sets that have the same name and type. Values
+	//     other than 60 seconds (the TTL for load balancers) will change the effect
+	//     of the values that you specify for Weight.
+	TTL *int64 `json:"tTL,omitempty"`
```

Comment:
This should be generated as `ttl` ... maybe the regex is messed up here: https://github.com/aws-controllers-k8s/pkg/blob/main/names/names.go#L167

Reply:
Bumping the code-generator version could resolve this

Reply:
I'll work on fixing this... i'll ping you once we have a fix merged in code-generator